src/config/config.ts

```diff
@@ -26,10 +26,15 @@ export const infuraRpcUrl = `https://celo-mainnet.infura.io/v3/${infuraApiKey}`;
 
 const rpcUrl = process?.env?.NEXT_PUBLIC_RPC_URL || celo.rpcUrls.default.http[0];
 
-const isMainnet = ['mainnet', celo.rpcUrls.default.http[0]].includes(rpcUrl);
+const isMainnet = [
+  'mainnet',
```

Comment:
im always confused as to weather mainnet is ethereum L1, or celo

Reply:
celo, but this config is too complex as it is I'd like to refactor it. I made it this way only so I could consider my archive node as mainnet (which it is) but this isn't good